opds.py

```diff
@@ -558,7 +558,7 @@ def make_link(ep, is_default):
                 feed, make_link, entrypoints, facets.entrypoint
             )
 
-        cls.add_breadcrumb_links(feed, lane, annotator)
+        cls.add_breadcrumb_links(feed, lane, facets.entrypoint)
```

Comment:
did you mean to change this to `feed.add_breadcrumb_links`? if not, why didn't this one change?

Reply:
I did mean to, and it's weird that it works without the change, but i guess cls.method(instance) is the same as instance.method().